static/components/shuttles-map.html

```diff
@@ -185,6 +199,24 @@
               icon: "static/images/shuttle.png",
               title: updates[i]["vehicleID"]
             });
+            
+            // Get cardinal direction for info window
+            var cardinal_dir = this.cardinalDirection(updates[i]["heading"]);
+            // Create info window and open it when shuttle clicked
+            (function(marker, i) {
+              /* Include ID, heading, speed, time in the infowindow */
+              var contentString = '<b>Shuttle ID: ' + updates[i]["vehicleID"] + '</b><br />' +
```

Comment:
String interpolation is a better way to constructing user-facing strings: https://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Literals#Interpolation